components/measurement/PerformanceDetails.js

```diff
@@ -19,24 +20,29 @@ const PerformanceDetails = ({
     },
     {
       label: intl.formatMessage({ id: 'Measurement.Details.Performance.Label.MaxPing' }),
-      value: maxPing.toString() + ' ms'
+      value: isNdt7 ? `~${maxPing.toString()} ms` : `${maxPing.toString()}ms`
```

Comment:
Maybe this can be reduced to `${isNdt7 ? '~' : ''}${maxPing.toString()} ms`

Reply:
Yes sure